src/server.js

```diff
@@ -55,6 +55,13 @@ const getCORSOptions = () => {
 // Custom get secrets handler
 const getSecretsHandler = {
 	get: function (target, prop, receiver) {
+		if (prop === 'toJSON') {
+			// Handle the toJSON case
+			return () => target;
+		}
+		if (Object.keys(target).length === 0) {
+			return undefined;
+		}
```

Comment:
Wouldnt this be a problem if the secrets object is empty `{}` but the hooks package does `secrets.toString()` or `JSON.stringify(secrets)`?

I don't think we should do this. It will not throw an error if someone uses a secret which is not mentioned as well.

Check this screenshot

![image](https://github.com/user-attachments/assets/4c132d96-5190-4acf-b992-2eb0e2aae16a)

Reply:
@annavara - I have addressed the change, please help a look again.